Gemfile.lock

```diff
@@ -34,43 +34,43 @@ GEM
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.2)
+    i18n (1.8.3)
       concurrent-ruby (~> 1.0)
     json (2.3.0)
-    jwt (2.1.0)
-    jwt_signed_request (2.5.1)
-      jwt (>= 1.5.0, < 2.2.0)
+    jwt (2.2.1)
+    jwt_signed_request (2.5.4)
+      jwt (>= 1.5.0)
       rack
-    loofah (2.4.0)
+    loofah (2.6.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (1.0.0)
     mini_portile2 (2.4.0)
-    minitest (5.14.0)
-    money (6.13.6)
+    minitest (5.14.1)
+    money (6.13.7)
       i18n (>= 0.6.4, <= 2)
     nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
-    pagerduty (2.1.2)
+    pagerduty (3.0.0)
       json (>= 1.7.7)
-    rack (2.2.2)
+    rack (2.2.3)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    railties (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
     rake (13.0.1)
     thor (1.0.1)
     thread_safe (0.3.6)
-    tzinfo (1.2.6)
+    tzinfo (1.2.7)
```

Comment:
### [tzinfo](https://tzinfo.github.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.6 → 1.2.7

[_~~change-log~~, [source-code](https://github.com/tzinfo/tzinfo)_]





<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/tzinfo/tzinfo/compare/v1.2.6...v1.2.7).

These are the first 10 commits:
- (7a603da) [Update to Ruby 2.7.0.](https://github.com/tzinfo/tzinfo/commit/7a603da1baa30696e7603c5c95cc3b28dc4c1d07)
- (5f9b1cd) [`$VERBOSE = false` won't be worked since `rb_warning` is cha…](https://github.com/tzinfo/tzinfo/commit/5f9b1cd56f80b8975dbe50678d11b14811a38167)
- (83bdd0e) [Fix erroneous 'wrong number of arguments' errors on JRuby 9.…](https://github.com/tzinfo/tzinfo/commit/83bdd0ed2a701ba85d1cb36cb11833385fb02b44)
- (e0e66f8) [Fix test failures on Ruby 1.8.7.](https://github.com/tzinfo/tzinfo/commit/e0e66f8468f20c61882752b8f8c694825683bc48)
- (1d57e47) [Don't rely on lexicographic version comparisons.](https://github.com/tzinfo/tzinfo/commit/1d57e4787297c94849189dc76da3f85957bfcc44)
- (ae54281) [Fix comments relating to taint/untaint removal.](https://github.com/tzinfo/tzinfo/commit/ae54281247c8e07f913090c5942c5be019d781c9)
- (62d902f) [Test for just the non-existence of #untaint.](https://github.com/tzinfo/tzinfo/commit/62d902fc6668d9445ad4b4ec251d9281a4429adb)
- (d8556f0) [Merge pull request #112.](https://github.com/tzinfo/tzinfo/commit/d8556f0ec2f4d152d6c874ecb5da5d6d60b5747a)
- (075d5e3) [Update to JRuby 9.2.11.0.](https://github.com/tzinfo/tzinfo/commit/075d5e31b6c5ab7e53679bfe1cb899790378d8dd)
- (1f81e20) [Use https for links where available.](https://github.com/tzinfo/tzinfo/commit/1f81e20ad8b2ea76a1012e666d98d5b8e9648239)


</details>